packages/a11y-testing/etc/a11y-testing.api.md

```diff
@@ -0,0 +1,146 @@
+## API Report File for "@fluentui/a11y-testing"
```

Comment:
This file was generating by running command in the package folder:
yarn update-api

and then manually copied from temp folder to this etc folder

Reply:
`build` will give you the message about manually copying, but `update-api` should copy automatically. Also if this isn't intended to be a published package we can just turn off API Extractor.